src/services/MapBounds/MapBounds.js

```diff
@@ -110,3 +111,26 @@ export const boundsWithinDegrees = function(bounds1, bounds2, maxAllowedDegrees)
                Math.abs(normalizedBounds1.getNorth() - normalizedBounds2.getNorth()),
                Math.abs(normalizedBounds1.getSouth() - normalizedBounds2.getSouth())])
 }
+
+
+/**
+ * Calculates the best bounding box to incorporate all the given coordinates.
+ */
+export const calculateBoundingBox = function(coordinates) {
```

Comment:
Is there an advantage to this over the turf `bbox` function used elsewhere? Leaflet can also produce bounding boxes.